index.html

```diff
@@ -48,25 +47,13 @@ <h2 class="subtitle">
         </div>
     </div>
 
-    <div class="box hero is-info has-text-centered-mobile is-vcentered">
-        <h1 class="title">
-            Webxdc can be anything
-        </h1>
-        <p>
-            Collaborative Tools as <br class="is-hidden-tablet">Editors, Todo-Lists<br>
-            Shared Calendars, Doodles<br>
-            Polls<br>
-            Games, of course
-        </p>
-    </div>
-
     <div class="box hero is-success">
         <div>
             <h1 class="title">
-                Try it out!
+                try it out, it's fun :) 
             </h1>
             <p>
-                Download <code>.xdc</code> from below and attach it to any chat in <a href="https://get.delta.chat">Delta Chat 1.30 or newer</a>.
+                you need to install and configure <a href="https://delta.chat">delta chat</a> and then download a <code>.xdc</code> file from below and attach it as a file to a chat and hit the "Start" button.
```

Comment:
too many "and", maybe this is better:

```suggestion
                install and configure <a href="https://delta.chat">delta chat</a>, download <code>.xdc</code> from below, attach it as a file to a chat and hit "start".
```